CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Changed
 - Migration to ElasticSearch 6.3
 - Blob export to LinShare
+- Native DKIM support for outgoing mails. The mailets from james-jdkim have been included in main James project.
```

Comment:
This is specific for Guice but did you do it for Spring?

Reply:
signing doesn't require injection

Reply:
```
benwa@horizon ../apache/onssd/james-project/server/app (git)-[master] % mvn dependency:tree | grep dkim                              
mvn dependency:tree  14,23s user 0,50s system 396% cpu 3,716 total
grep --color=auto dkim  0,02s user 0,02s system 1% cpu 3,716 total
```

I suppose spring users can not use it :-(

Reply:
@chibenwa : I don't understand the output of your grep.

Reply:
server/app does not depend on james-server-mailet-dkim which means the mailet is not usable currently on top of Spring. Either you add the dependency - and test it - or you recognise that this is a guice only feature.

Reply:
Ok, good point, spring doesn't have the dependency.
However, requiring to test it is stricter than our current policy. By now, if smoke tests work, we are happy enough, right?
I'll add the dependency to this PR.